mousestyles/dynamics/__init__.py

```diff
@@ -431,3 +431,97 @@ def get_score(true_day, simulated_day, weight=[1, 10, 50, 1]):
             score += weight[int(status)]
     score = score/len_true
     return score
+
+
+def find_best_interval(df, strain_num, interval_length_initial=np.arange(600,
+                       7800, 600)):
+    r"""
+    Returns the optimized time interval length and the corresponding
+    fake mouse behavior string with the evaluation score for a
+    particular mouse strain. The input data are the pandas DataFrame
+    generated by function create_time_matrix which contains all strains
+    information, the strain number that we want to develop, and some
+    initial time interval length we want to optimize on. The outputs
+    are the optimized time interval length, the similated mouse states
+    string using that optimized time interval and the evaluation score
+    comparing this simulated mouse with the real mice behavior in the
+    strain using the same optimized time interval length.
+
+    Parameters
+    ----------
+    df: Pandas.DataFrame
+        a huge data frame containing info on strain, mouse
+        no., mouse day, and different states at chosen time
+        points.
+    starin_num: int
+        an integer specifying the desired mouse strain
+    interval_length_initial: numpy.ndarray
```

Comment:
For clarification, can you add brief explanations why you set the default of interval_length_initial as (600, 1200, ..., 7800)?